### PR TITLE
Added export support for account, instance level vdrs

### PR DESCRIPTION
**Pull request by ApoorvaTiwari**
Supports download of vdrs for accounts and instance level in addition with organization level.

### CODE REVIEW

File: src/core/saveCommonResources.js
Comment (nick-bair) [inline src/core/saveCommonResources.js]: @ApoorvaTiwari can you remove the commented out code if it's not needed?

**Review by nick-bair — APPROVED** [review]
Looks good! Thanks :)

**Comment by Jayme-T** [comment]
@ApoorvaTiwari  this looks good thanks- Please upgrade the package json version and let me know if you want me to merge this or if we were holding off for additional changes

**Comment by Jayme-T** [comment]
Oh and we will need to pull in master because Mike made some changes and those got merged

**Review by Jayme-T — CHANGES_REQUESTED** [review]
Hey Apoorva can we submit a version change and pull master into this branch. Then let me know if you want it merged right away or not. Thanks

**Comment by ApoorvaTiwari** [comment]
Hey Jamie, Pradheep is working on doctor as a service and asked me to hold this until that gets released.

**Comment by GiriNellore** [comment]
As we handled this with Doctor service changes, closing this PR.

**Pull request closed without merging**
